### PR TITLE
App rebranding and components refactoring

### PR DESCRIPTION
**Pull request by yvesfracari**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
-  subsets: ["latin"],
-  variable: "--font-sans",
-});
-
 export const metadata: Metadata = {
   title: "Create Next App",
   description: "Generated by create next app",
 };
 
+const flechaS = localFont({
```
Comment (ribeirojose) [inline src/app/layout.tsx:14]: any reason for us to be using the same branding from CoW AMM here? 🤔

File: src/components/AlertCard.tsx [hunk 2]
```diff
@@ -15,18 +15,18 @@ export function AlertCard({
     <div className="max-w-md" role="alert">
       <div
         className={cn(
-          "font-bold text-slate12 rounded-t px-4 py-2 mt-1",
-          style === "error" ? "bg-tomato9" : "bg-amber9",
+          "font-bold text-slate12 px-4 py-2 mt-1",
+          style === "error" ? "bg-tomato9" : "bg-amber9"
```
Comment (ribeirojose) [inline src/components/AlertCard.tsx:19]: prefer theme variable colors

File: src/components/Dialog.tsx [hunk 3]
```diff
@@ -9,65 +18,63 @@ export function Dialog({
...
+              }}
+            >
+              {submitText}
+            </Button>
           </div>
-          <DialogPrimitive.Close asChild>
+          <DialogClose asChild>
             <button
-              className="absolute right-[10px] top-[10px] inline-flex size-[30px] items-center justify-center text-slate12 hover:font-black focus:shadow-[0_0_0_2px]	focus:shadow-slate2 focus:outline-none"
+              className="absolute right-[10px] top-[10px] inline-flex size-[30px] items-center justify-center text-sand12 hover:font-black focus:outline-none"
```
Comment (ribeirojose) [inline src/components/Dialog.tsx:70]: prefer theme variable colors

File: src/components/SelectInput.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}: {
+  label?: string;
+  name: string;
+  options: { id: number | string; value: string }[];
+  onValueChange: (value: string) => void;
+  placeholder?: string;
+  tooltipText?: string;
+}) => (
+  <div className="flex w-full flex-col justify-start">
+    <label className="block text-sm text-sand1" htmlFor={name}>
```
Comment (ribeirojose) [inline src/components/SelectInput.tsx:23]: prefer theme variable colors

**Comment by ribeirojose** [comment]
here's some general feedback:

- since this is an open source project, IMO we should set `hideAttribution: true` in the react-flow component at least for now, or move it to the bottom left corner if we can, it's a bit intrusive right now;
- since this is a UI PR, pushing a screenshot would've made it easier to review/understand at first what's going on here;

and specifics:

Screenshot 2024-03-27 at 10 29 54 AM
- would remove shadow from checkbox as it's not part of the visual language we created across the app;

Screenshot 2024-03-27 at 10 28 07 AM
- remove shadow from the react-flow component's borders;

Screenshot 2024-03-27 at 10 27 51 AM
- the margin in the hovered element here is a bit weird IMO. Either we should have no margin or have a padding in the hovered element itself so that the hover background goes from edge to edge.

Screenshot 2024-03-27 at 10 27 22 AM
- I think this component broke during the refactoring 😬 

Screenshot 2024-03-27 at 10 27 17 AM
- same for this one

approved conditionally pending working on these items

**Comment by ribeirojose** [comment]
one more thing: I missed some buttons, and noticed they're some hidden because of the styling  😛 see attached

Screenshot 2024-03-27 at 10 42 59 AM
Screenshot 2024-03-27 at 10 42 54 AM

File: src/app/layout.tsx (hunk 1 above)
Comment (yvesfracari) [reply src/app/layout.tsx:14]: we are just using for now, just bc it was a theme that was already defined. Maybe CoW team can help us on this.

File: src/components/AlertCard.tsx (hunk 2 above)
Comment (yvesfracari) [reply src/components/AlertCard.tsx]: Notice that this component isn't being used. I will delete it.

**Pull request merged**